### ISSUE TITLE
Problem with bower

### ISSUE THREAD

**Issue by rkeshmir** [danielbruce/entypo#29]
Please add this repository to bower.io so that we can install it with bower

**Comment by g00glen00b** [comment]
You should already be able to install it with Bower, though it will not pop up in the online search results:

```
g00glen00b$ bower search entypo
Search results:

    entypo git://github.com/danielbruce/entypo.git
    entypo-plus git://github.com/chancancode/entypo-plus.git
    entypo-css-only git://github.com/g00glen00b/entypo.git
```

Also, Bower allows you to directly install any Git repository with it.

However, they could indeed improve the support by adding a bower.json file and adding ignored files, tagging builds, ... .

**Comment by rkeshmir** [comment]
So I should type `bower install entypo --save` ?

**Comment by g00glen00b** [comment]
Yeah, it will work, but like I said before, they could improve it by tagging builds and ignoring files.

You can either use:

```
bower install entypo
bower install danielbruce/entypo
bower install git://github.com/danielbruce/entypo.git
```

Anyways, every repository that can be found on Github (or any other Git repository) can be used within Bower.
